tests/modules/introductions/consent-workflow-repository-drizzle.integration.test.ts

```
@@ -0,0 +1,169 @@
+				{
+					workflowId: workflow.id,
+					candidateUserId: inactiveCandidate.id,
+					position: 2,
+				},
+			],
+		});
+
+		await workflowRepository.updateStatus({
+			workflowId: workflow.id,
+			status: "consent_requested",
+			fields: { activeAttemptId: attempts[0]!.id },
```

Biome (2.4.4)

[error] Forbidden non-null assertion.

(lint/style/noNonNullAssertion)